src/test/java/org/zendesk/client/v2/RealSmokeTest.java

```diff
@@ -1583,6 +1584,21 @@ public void createOrganizationMemberships() throws Exception {
         }
     }
 
+    @Test
+    public void lookupOrganizationByExternalId() throws Exception {
+        createClientWithTokenOrPassword();
+
+        Organization newOrganization = newTestOrganization();
+        newOrganization.setExternalId("i");
+        Organization resultOrganization = instance.createOrganization(newOrganization);
+        assertNotNull(resultOrganization);
+
+        Iterable<Organization> or =  instance.lookupOrganizationsByExternalId("i");
+        assertTrue(or.iterator().hasNext());
```

Comment:
Only makes sure something is returned, we should also make sure the size is ok, what about the following?

```suggestion
        assertEquals(1, StreamSupport.stream(or.spliterator(), false).count());
```

Reply:
Added the change in the test.